src/generators/web.cr

```diff
@@ -189,6 +199,14 @@ class LuckyCli::Generators::Web
           version: ~> 0.7.3
       DEPS_LIST
     end
+
+    if with_sec_tester?
+      append_text to: "shard.yml", text: <<-DEPS_LIST
+        lucky_sec_tester:
```

Comment:
oops... just realized if you do api_only and with_sec_tester, it'll append this to dependencies and not development dependencies... My comment [here](https://github.com/luckyframework/lucky_cli/issues/735#issuecomment-1073047223) on just requiring yaml is starting to sound a lot nicer....

Reply:
Alternatively, we could forcibly add a devdep section and use "append before" and "append after" to add dev deps vs dev deps